packages/protocol/contracts/stability/StableToken.sol

```diff
@@ -265,7 +271,7 @@ contract StableToken is
    */
   function burn(uint256 value)
     external
-    onlyRegisteredContract(EXCHANGE_REGISTRY_ID)
```

Comment:
How will this be backwards compatible for cUSD?

Reply:
forgot the fallback behavior

Reply:
solved here: https://github.com/celo-org/celo-monorepo/pull/6780/files#diff-d3d584105569d60895c3dee70920dfd51d712edd2fef038fa6a5e7e82e3fc18dR397